### PR TITLE
My conference website

### PR DESCRIPTION
**Pull request by mbdesigns1989**
## HTML/CSS & JavaScript capstone project - Conference page
I used Flexbox to place elements on the page.
I created UIs adaptable to different screen sizes using media queries.
I used GitHub Pages to deploy web pages.
I used JavaScript to manipulate DOM elements.
I used JavaScript events.
I used objects to store and access data.

### CODE REVIEW

**Comment by mbdesigns1989** [comment]
Please, I repeated week one so every review counts, I have been working on the project since Saturday, I don't want to repeat the module and all my work for 
the past 5 weeks were for nothing, so please if you're going to ask for changes help me and give me hints or refer me to links to help apply the changes, thanks.

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello  @mbdesigns1989 

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+    integrity="sha384-SZXxX4whJ79/gErwcOYf+zWLeJdY/qpuqC4cAa9rOGUstPomtqpuNWT9wdPEn2fk" 
+    crossorigin="anonymous">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700&display=swap">
+</head>
+<body>
+    <header class="header">
+        <nav class="nav">
+            <div class="icon" id="bars">
+                <i class="fas fa-bars"></i>
+            </div>
```
Comment (mikenath223) [inline index.html:19]: - [ ] There should be a navbar menu on the mobile screen but it doesn't appear on yours. Please fix this both on the homepage and the about page.

| Original | Yours |
| ----- | ------ |
|  |  |

File: index.html [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+                </a>
+            </div>
+            <ul class="top-nav">
+                <li class="top-nav__item">
+                    <a href="#" class="top-nav__link">
+                        <i class="fab fa-twitter"></i>
+                    </a>
+                </li>
+                <li class="top-nav__item">
+                    <a href="#" class="top-nav__link">
```
Comment (mikenath223) [inline index.html:32]: - [ ] On the mobile screen there is a white border existing around the screen, kindly remove it and ensure the page content fills the entire screen. Please fix it both on the homepage and the about page.


| Original | Yours |
| ----- | ------ |
|  |  |

File: index.html [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+            </div>
+        </section>
+        <section>
+            <div class="container">
+              <div class="speakers">
+                  <h2 class="speakers-heading">Featured Speakers</h2>
+                    <ul class="speakers-list" id="speakers-list">
+                        
+                    </ul>
+               </div>
```
Comment (mikenath223) [inline index.html:155]: - [ ] In the featured speakers section there is a background seen behind all images, kindly include it.

| Original | Yours |
| ----- | ------ |
|  |  |

File: index.html [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+        </section>
+        <section>
+            <div class="container bg-t">
+                <div class="partner">
+                    <h2 class="partner-heading">Partner</h2>
+                    <ul class="partners-list">
+                        <li class="partner-list-item">
+                            <i class="fab fa-facebook"></i>
+                        </li>
+                        <li class="partner-list-item">
```
Comment (mikenath223) [inline index.html]: - [ ] Kindly include the images in the partners section as seen in the original.
- [ ] Also, ensure you use the exact background color, you can use a color picker extension to get the right color.

| Original | Yours |
| ----- | ------ |
|  |  |

File: index.html [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+                                    DMC Ignite
+                                </h2>
+                                <p class="program-list-item-container__text">
+                                    Get opporunites to network with DMC affiliates around the world, also after the summit.
+                                </p>
+                            </div>
+                        </li>
+                    </ul>
+                    <button class="btn btn-large" type="button">SEE THE WHOLE PROGRAM</button>
+                </div>
```
Comment (mikenath223) [inline index.html]: - [ ] The `See the whole program` element is simply a text in the original design not a button with a red background, kindly fix it.

| Original | Yours |
| ----- | ------ |
|  |  |

File: about.html [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+                        <li class="partner-list-item">
+                            <i class="fab fa-twitter"></i>
+                        </li>
+                        <li class="partner-list-item">
+                            <i class="fab fa-amazon"></i>
+                        </li>
+                        <li class="partner-list-item">
+                            <i class="fab fa-airbnb"></i> 
+                        </li>
+                        <li class="partner-list-item">
```
Comment (mikenath223) [inline about.html]: - [ ] On the original design the about page doesn't have a partners section please remove it.

| Original | Yours |
| ----- | ------ |
|  |  |

File: about.html [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+            <div class="footer">
+                <div class="logo">
+                    <a  href="index.html">
+                       <h2>CGDMC.2021</h2>  
+                    </a>
+                </div>    
+                <p class="rights">
+                    @ Cairo International Convention Center.
+                </p>
+            </div>
```
Comment (mikenath223) [inline about.html]: - [ ] On the footer in the about page, kindly ensure you use the exact background color as seen in the original.

| Original | Yours |
| ----- | ------ |
|  |  |

**Comment by mbdesigns1989** [comment]
please, I did everything I can, if you ask for changes please help me and give me hints to help solve the problem, thanks.

**Review by mimipeshy — APPROVED** [review]
## Hi @mbdesigns1989  👋🏾 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

**Pull request merged**